core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.ts

```diff
@@ -131,7 +131,7 @@ export class PracticeTabComponent implements OnInit {
       PracticeSessionPageConstants.PRACTICE_SESSIONS_URL, {
         topic_url_fragment: this.topicUrlFragment,
         classroom_url_fragment: this.classroomUrlFragment,
-        comma_separated_subtopic_ids: selectedSubtopicIds.join(',')
+        stringified_subtopic_ids: JSON.stringify(selectedSubtopicIds)
```

Comment:
This modification is done because we want to normalize arguments while schema validation process in the backend.
Some of the params already use JSON.stringify so modifying here also to send list of ids in string format instead of comma seperated.
Ex: ```'id_1,id_2'``` --------------> ```'["id_1", "id_2"]'```